ganttproject/src/net/sourceforge/ganttproject/ResourceDefaultColumn.java

```diff
@@ -53,6 +55,14 @@ private ResourceDefaultColumn(ColumnList.Column delegate, Class<?> valueClass, S
     myValueClass = valueClass;
   }
 
+  public Comparator<?> getSortComparator() {
```

Comment:
Is it used anywhere besides the sorting code itself? Can we just keep a local comparator instance in that code, without storing the reference here?